utils/frame/remote-externalities/src/lib.rs

```diff
@@ -412,7 +415,7 @@ impl<B: BlockT + DeserializeOwned> Builder<B> {
 	) -> Result<Vec<KeyValue>, &'static str> {
 		let now = std::time::Instant::now();
 		let keys = self.rpc_get_keys_paged(prefix, at).await?;
-		let uri = Arc::new(self.as_online().transport.uri.clone().unwrap());
+		let client = self.as_online().transport.remote_client.clone().unwrap();
```

Comment:
I actually tried this, but even the local tests would then stall. Maybe it was because of lagging this `#[tokio::test(flavor = "multi_thread")]`?

Reply:
it worked for the tests in remote externalities but didn't try it in try-runtime